### PR TITLE
Add contribute.json file.

### PR DESCRIPTION
**Pull request by Osmose**
Add a contribute.json file a la http://contribute.paas.allizom.org/.
I've been adding these to the repos of Webdev projects. If you're
unsure if you want one, I can answer any questions you have, or you
can reject the PR if you don't want one. Thanks!

### CODE REVIEW

**Comment by Callek** [comment]
@Osmose so I'm not an admin of wikimo's code here, but I am very curious on this setup. Can you e-mail me at jwood@mozilla.com with further details. Such as does this feed into anything like http://www.joshmatthews.net/bugsahoy/ or http://www.joshmatthews.net/trainingmontage/ or anything like it.

Does it pull from (Mozilla's)? github projects automatically, etc?

My use case is for Release Engineering (we have a lot of stuff!)

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "wiki.mozilla.org",
+    "description": "Official public wiki of the Mozilla Project.",
+    "repository": {
+        "url": "https://github.com/mozilla/wiki.mozilla.org",
+        "license": "MPL 2.0",
+        "tests": "https://travis-ci.org/mozilla/airmozilla/"
```
Comment (Callek) [inline contribute.json]: well we're certainly not airmozilla here :-)

Comment (Osmose) [inline contribute.json]: SHHHHHHHHH YOU SAW NOTHING

**Comment by christi3k** [comment]
This generally looks fine.

Question, though, is contribute.json supposed to be web-accessible in production? e.g. wiki.mozilla.org/contribute.json?

**Comment by Osmose** [comment]
> Question, though, is contribute.json supposed to be web-accessible in production? e.g. wiki.mozilla.org/contribute.json?

That's optional. The spec says (and http://mozilla.github.io/webdev/ assumes) it should be at the root of the repo but it's common to put it at the root of the domain for websites as well.

**Comment by gozer** [comment]
Implemented a long while ago in b8b37695f61ddde730facc6f59ed0c325b0aac87#diff-85f8f9ffd2726edd8861b2e723aadb72

**Pull request closed without merging**